--- config.mak	2006-10-15 16:52:23.000000000 +0200
+++ config.mak	2006-10-15 16:52:34.000000000 +0200
@@ -6,7 +6,7 @@
 ARCH=X86
 SYS=MACOSX
 CC=gcc
-CFLAGS=-Wall -I. -O4 -ffast-math -D__X264__ -falign-loops=16 -DHAVE_MMXEXT -DHAVE_SSE2 -DARCH_X86 -DSYS_MACOSX -DHAVE_PTHREAD -fPIC -s -fomit-frame-pointer
+CFLAGS=-Wall -I. -O4 -ffast-math -D__X264__ -falign-loops=16 -DHAVE_MMXEXT -DARCH_X86 -DSYS_MACOSX -DHAVE_PTHREAD -fPIC -s -fomit-frame-pointer
 LDFLAGS= -lm -lmx -lpthread -s
 AS=nasm
 ASFLAGS=-O2 -f macho -DPREFIX -D__PIC__
